cfg/pgrapher/experiment/icarus/sp.jsonnet

```diff
@@ -61,11 +62,11 @@ function(params, tools, override = {}) {
       use_multi_plane_protection: false,
       mp3_roi_tag: 'mp3_roi%d' % anode.data.ident,
       mp2_roi_tag: 'mp2_roi%d' % anode.data.ident,
-      
+
       isWrapped: false,
       process_planes: [0, util.anode_split(anode.data.ident)], // balance the left and right split
 
     } + override,
-  }, nin=1, nout=1, uses=[anode, tools.field] + pc.uses + spfilt),
+  }, nin=1, nout=1, uses=[anode, eresponse, etools.field] + pc.uses + spfilt),
```

Comment:
Instead of "etools.field", I think it should be "tools.field"?

Reply:
You are absolutely right, good catch